plugins/obs-filters/CMakeLists.txt

```diff
@@ -20,20 +20,31 @@ include_directories(${LIBSPEEXDSP_INCLUDE_DIRS}
 
 set(obs-filters_SOURCES
 	obs-filters.c
-	color-correction-filter.c
 	async-delay-filter.c
-	gpu-delay.c
-	crop-filter.c
-	scale-filter.c
-	scroll-filter.c
 	chroma-key-filter.c
-	color-key-filter.c
+	color-correction-filter.c
 	color-grade-filter.c
-	sharpness-filter.c
+	color-key-filter.c
+	compressor-filter.c
+	crop-filter.c
+	crosshatching-filter.c
+	cross-stitching-filter.c
+	droste-filter.c
 	gain-filter.c
-	noise-gate-filter.c
+	gpu-delay.c
+	like-a-dream-filter.c
+	luma-key-filter.c
 	mask-filter.c
-	compressor-filter.c)
+	noise-gate-filter.c
+	night-vision-filter.c
+	noise-gate-filter.c
```

Comment:
noise-gate-filter.c is listed twice:  here, and line 38.  This one is alphabetically correct.